player/droplayer.hpp

```diff
@@ -69,6 +69,7 @@ struct DRO_HEADER
 struct DRO_PLAY_OPTIONS
 {
 	UINT8 v2opl3Mode;	// DRO v2 DualOPL2 -> OPL3 fixes
+	UINT32 playbackSpeedScale; // Set to 0x10000 for 1.0 speed, or 16.16 fixed point
```

Comment:
The option is consistent across all players, so I'd prefer to not have it copy-pasted between all player-specific configurations.

I think I'll make a common `PLAY_OPTIONS` struct later that just gets included everywhere and includes other things (sample rate, callbacks) as well.